bom/openhab-addons/pom.xml

```
@@ -117,6 +117,11 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.anthem</artifactId>
+      <version>${project.version}</version>
+    <dependency>
+    </dependency>
```

This will probably fix the build:
```suggestion
    </dependency>
    <dependency>
```

---

Oh my. Yes I'm sure it will.

---

It failed again, but for another reason. You also need to add your binding to this POM:
https://github.com/openhab/openhab-addons/blob/9a05e9f3b5662ecb9701e95ecb02e07bae37d98c/bundles/pom.xml#L58-L59

---

Ugh. Sadly it's been so long since I've written a binding I've forgotten all the places that need to be touched. 😢

---

But now you got them all - build succeeded. 🙂